crates/examples/examples/chain_spec_runtime_kusama.rs

```diff
@@ -0,0 +1,117 @@
+use std::path::PathBuf;
+
+use anyhow::{anyhow, Context, Result};
+use serde_json::{Map, Value};
+use tempfile::Builder;
+use zombienet_sdk::{NetworkConfigBuilder, NetworkConfigExt};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt::init();
+
+    let temp_dir = Builder::new()
+        .prefix("zombienet-chain-spec-")
+        .tempdir()
+        .context("creating temporary directory")?;
+
+    let temp_path = temp_dir.path().to_path_buf();
+
+    let kusama_runtime_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("../orchestrator/testing/kusama_runtime-v2000002.compact.compressed.wasm");
```

Comment:
I think we can get these from the release page directly without have them in our repo.